### ISSUE TITLE
Issue with the orientation of the model

### ISSUE THREAD

**Issue by BiomechatronicsRookie** [Walter0807/MotionBERT#79]
Hi! I ran into an issue when trying to get the pose of some videos. Even if the camera was leveled (completely vertically) the estimates I get seem to be skewed in the direction of the "z" axis (off the screen, maybe the attached picture shows it better). Is this a smotheing anyone has encountered before?

<img width="445" alt="image" src="https://github.com/Walter0807/MotionBERT/assets/67059763/dc7a1884-ab5c-4352-b261-9a715222fe3a">

The vertical axis in this image would be the z axis of the output of the network.

**Comment by Walter0807** [comment]
Can't understand the meaning of this figure, could you please explain further?

**Comment by BiomechatronicsRookie** [comment]
Yeah sorry for the little information, I understand it was not enough.

So this image is a lifted pose from a recording in which the person was sideways on the image, as in one side of the body was partially occluded. The person on the image was standing still, however, in the plot attached previously, you can see how the points corresponding to the ankle position (the one on the far most right if the image) and the point corresponding to the head (on the other side) seem to show that the person was tilted towards the phone (the xy plane, the top axes show them). The same happens from different angles. 

I would expect (maybe) the lifted pose not to be tilted if the person is standing still. Is this seen in other cases?

I hope I explained myself properly

**Comment by whatishisname** [comment]
you just have to invert the axis when you plot the results.
I did encounter a similar problem when trying to plot the training data.
I had to invert the y-axis using plt.gca().invert_yaxis() (matplotlib) and that means the (0,0) of the plot is located at the left top corner not the bottom left corner

![plot](https://github.com/Walter0807/MotionBERT/assets/144952075/5e2a8adf-d759-40a6-9994-68de7dc4cfeb)

**Comment by Walter0807** [comment]
Closing due to inactivity. Feel free to reopen if needed.
